### PR TITLE
Shadow foundation updates

### PR DESCRIPTION
**Pull request by amritadesmet**
In relation to #4467

### CODE REVIEW

**Review by Fercas123 — APPROVED** [review]

File: site/docs/foundations/shadow.mdx [hunk 1]
```diff
@@ -19,4 +26,48 @@ There are six shadow elevations to choose from.
 | `--salt-shadow-400` | 0           | 6           | 10        |
 | `--salt-shadow-500` | 0           | 12          | 40        |
 
+## Scrollable area: Shadow-100
```
Comment (joshwooding) [inline site/docs/foundations/shadow.mdx:29]:
```suggestion
## Scrollable area: shadow-100
```

File: site/docs/foundations/shadow.mdx [hunk 2]
```diff
@@ -19,4 +26,48 @@ There are six shadow elevations to choose from.
...
+
+Shadow-100 is the closest to the surface and is used exclusively for scrollable areas where the content exceeds the visible screen space, allowing users to scroll up or down. This is typically positioned at the top of an overflowing area, providing a subtle hint of depth without casting a noticeable shadow.
+
+## Region: Shadow-200
```
Comment (joshwooding) [inline site/docs/foundations/shadow.mdx:33]:
```suggestion
## Region: shadow-200
```

File: site/docs/foundations/shadow.mdx [hunk 3]
```diff
@@ -19,4 +26,48 @@ There are six shadow elevations to choose from.
...
+
+Shadow-200 introduces a slight elevation, offering a soft shadow that begins to separate elements from the background. This level is suitable for components that require a gentle lift, such as cards or small panels.
+
+<Image
+  src="/img/foundations/shadow-container.png"
+  alt="Diagram showing a region shadow on a container."
+/>
+
+## Hover: Shadow-300
```
Comment (joshwooding) [inline site/docs/foundations/shadow.mdx:42]:
```suggestion
## Hover: shadow-300
```

File: site/docs/foundations/shadow.mdx [hunk 4]
```diff
@@ -19,4 +26,48 @@ There are six shadow elevations to choose from.
...
+
+Shadow-300 provides a moderate level of separation with a more pronounced shadow. It is ideal for elements that need to stand out more distinctly, such as dropdown menus or tooltips.
+
+## Pop-out: Shadow-400
```
Comment (joshwooding) [inline site/docs/foundations/shadow.mdx:46]:
```suggestion
## Pop-out: shadow-400
```

File: site/docs/foundations/shadow.mdx [hunk 5]
```diff
@@ -19,4 +26,48 @@ There are six shadow elevations to choose from.
...
+
+Shadow-400 offers a strong contrast with well-defined edges, making it perfect for components that require significant emphasis such as a popout. A popout is a window above the interface with a shadow that temporarily overlays elements, e.g. Toast, Tooltip, Overlay.
+
+<Image
+  src="/img/foundations/shadow-popout.png"
+  alt="Diagram showing a pop-out shadow."
+/>
+
+## Modal: Shadow-500
```
Comment (joshwooding) [inline site/docs/foundations/shadow.mdx:55]:
```suggestion
## Modal: shadow-500
```

File: site/docs/foundations/shadow.mdx [hunk 6]
```diff
@@ -19,4 +26,48 @@ There are six shadow elevations to choose from.
...
+
+Shadow-500 is the furthest away from the surface and is reserved for modal dialogs only. It creates a dramatic effect with a strong shadow, ensuring that modal dialogs are clearly distinguished from the rest of the content. For a modal, the user must interact with the overlay window, which typically has a scrim behind it, before interacting with the page content.
+
+<Image
+  src="/img/foundations/shadow-modal.png"
+  alt="Diagram showing a modal shadow."
+/>
+
+## Scrim
```
Comment (joshwooding) [inline site/docs/foundations/shadow.mdx:64]: Personally Scrim feels shoehorned into "Shadows". I guess at the moment there's nowhere else to put it...

File: site/docs/foundations/shadow.mdx (hunk 1 above)
Comment (joshwooding) [inline site/docs/foundations/shadow.mdx:29]: It feels weird that this whole page is about shadow ramps then starts immediately referring them to by use-case or semantics. Should the order of the heading be swapped?

Comment (amritadesmet) [reply site/docs/foundations/shadow.mdx]: Good point, will change this

File: site/docs/foundations/shadow.mdx (hunk 6 above)
Comment (amritadesmet) [reply site/docs/foundations/shadow.mdx:64]: Spoke with Ben and going to keep

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**